releng/release-notes-next/azure-linux.config

```diff
@@ -0,0 +1,2 @@
+Update the list of packages installed by default in the chroot to include shadow-utils so creating the chroot does not fail.
```

Comment:
Can you please amend to mention the name of chroot?

... in the chroot of Azure Linux to include ....

Reply:
Rephrased.